resources/patches/oe_bootstrap_theme_rename_bcl_toasts_dir.patch

```diff
@@ -0,0 +1,4 @@
+diff --git a/assets/bcl/bcl-toast/bcl-toasts.html.twig b/assets/bcl/bcl-toasts/bcl-toasts.html.twig
+similarity index 100%
+rename from assets/bcl/bcl-toast/bcl-toasts.html.twig
+rename to assets/bcl/bcl-toasts/bcl-toasts.html.twig
```

Comment:
I would prefer to see this fixed in `oe_bootstrap_theme`, so we don't have to merge with this patch.
Also I wonder about side effects for other things that use `toasts`.